hbw/app/helpers/hbw/task_helper.rb

```diff
@@ -0,0 +1,21 @@
+module HBW
+  module TaskHelper
+    def entity_url(task, entity_class)
+      entity_url_params = []
+
+      widget.config[:entities].fetch(entity_class)[:task_list][:entity_url_params].each do |param|
+        entity_url_params.append(task.send(param))
+      end
+
+      widget.config[:entities].fetch(entity_class)[:task_list][:entity_url] % entity_url_params
+    end
+
+    def entity_code_key(entity_class)
+      HBW::Widget.config[:entities].fetch(entity_class)[:entity_code_key]
```

Comment:
```suggestion
      widget.config[:entities].fetch(entity_class)[:entity_code_key]
```

Reply:
I agree with you If you point is consistency, but I prefer class method here.